src/lender/coordinator.sol

```diff
@@ -586,7 +584,7 @@ contract EpochCoordinator is Auth, Math, FixedPoint  {
         // assessor performs senior debt reBalancing according to new ratio
         assessor.changeSeniorAsset(seniorSupply, seniorRedeem);
         // the new reserve after this epoch can be used for new loans
-        assessor.changeReserveAvailable(newReserve);
+        assessor.changeBorrowAmountEpoch(newReserve);
         // reset state for next epochs
         lastEpochExecuted = epochID;
```

Comment:
I think we should call wipe here, in case there are funds sitting in the reserve that were not able to transferred to the mgr due to epoch submission period

Reply:
We should  set the flag `   submissionPeriod = false;` before we update the tranches.
Thats a way to guarantee that execute epoch can always `draw `or `wipe`